2.6.0/spec/subscriptions.md

```diff
@@ -1,11 +1,79 @@
-# Subscriptions endpoint
+# Subscriptions Endpoint
 
 Used to fetch a list of all active subscriptions from a CO belonging to the current SP.
 
 ## Methods
 
 ### GET
 
+#### Available services for a single Access
+
+To get available services for a single access use accessId as key. The response contains details about activated services and the services
+ available for activation.
+ 
+Request
+```http
+GET /onapi/2.6/services/{accessId} HTTP/1.1
+```
+
+Response 
+```HTTP
+HTTP/1.1 200 OK
+Content-Type: application/json
+```
+#### Example data from a single access response 
+
+When a single access is requested the response contains a single JSON-object containing the accessId and an array of subscriptions.
+
+```JSON
+{
+  "accessId": "8732c2f065e2490babce820e94b1011a",
+  "subscriptions": [
```

Comment:
I would still prefer a plain list of subscription objects here. If we want to make another filter based on for example `service` this structure would be awkward.